projects/plugins/social/src/js/classic-editor.js

```diff
@@ -0,0 +1,46 @@
+jQuery( function ( $ ) {
+	const state = jetpackSocialClassicEditorInitialState; // eslint-disable-line no-undef -- This is set by wp_localize_script().
+	const form = $( '#publicize-form' );
+
+	if ( state.sharesRemaining > state.numberOfConnections ) {
+		return;
+	}
+
+	const checkboxes = $( '#publicize-form' ).find( 'input[type="checkbox"]' );
+
+	// If we're all out of shares, disable all connections and call it a day.
+	if ( state.sharesRemaining === 0 ) {
+		checkboxes.each( function () {
+			$( this ).parent().addClass( 'wpas-disabled' );
+			$( this ).prop( 'disabled', true );
```

Comment:
I think we might also need to set `checked` to false here, as it seems like the connections are disabled in the UI, but the connection itself is enabled:
<img width="276" alt="image" src="https://user-images.githubusercontent.com/96462/189117776-0c88d622-e061-4693-8b6d-f539744099cf.png">

Reply:
Odd, that should be handled by the filter! Let me take a look.

Reply:
Ok ignore me, I can't replicate it. I must have had something cached.